snippets/csharp/System.Linq/Enumerable/AggregateTSource/enumerable.cs

```diff
@@ -705,18 +698,18 @@ static void ElementAtEx1()
         static void ElementAtOrDefaultEx1()
         {
             // <Snippet29>
-            string[] names =
-                { "Hartono, Tommy", "Adams, Terry", "Andersen, Henriette Thaulow",
-                    "Hedlund, Magnus", "Ito, Shu" };
+            string[] names = [ "Hartono, Tommy", "Adams, Terry", "Andersen, Henriette Thaulow",
+                "Hedlund, Magnus", "Ito, Shu" ];
 
-            int index = 20;
+            const int index = 20;
 
-            string name = names.ElementAtOrDefault(index);
+            var name = names.ElementAtOrDefault(index);
+            //var name = names[index];
 
             Console.WriteLine(
                 "The name chosen at index {0} is '{1}'.",
                 index,
-                String.IsNullOrEmpty(name) ? "<no name at this index>" : name);
+                name ?? "<no name at this index>");

```

Comment:
You should add a comment that the `??` operator is semantically different than `IsNullOrEmpty`